game/static/game/css/leaderboard.css

```
@@ -0,0 +1,316 @@
+    to { opacity: 1; transform: translateY(0); }
+}
+
+/* Table container for overflow scroll */
+.table-container {
+    width: 100%;
+    overflow-x: auto;
+    -webkit-overflow-scrolling: touch;
+    margin-top: 10px;
+    background: var(--card-bg, #16162a);
+    border-radius: 12px;
+    overflow: hidden;
```

Stylelint (17.13.0)

[error] Overridden property "overflow-x" by shorthand "overflow" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)